.github/config.yml

```diff
@@ -0,0 +1,18 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here! Be sure to follow the issue template!
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thanks for opening this pull request! Please check out our contributing guidelines.
```

Comment:
We could hyperlink the guidelines here (if it's supported)

Reply:
Somehow, they get automatically hyperlinked by github anyways. I think it's coded in the app that creates the message (if the contributing guidelines is present in the repo, that is).

Reply:
Well, the guidelines are linked when opening an issue. Since I already opened a PR, I can't check that.